id-repository/bioextractor-service/Dockerfile

```diff
@@ -44,12 +57,30 @@ ENV extraction_format_env=${extraction_format}
 
 ENV server_port_env=${server_port}
 
-ENV bio_sdk_folder_env=${bio_sdk_folder}
+ARG boisdk_local_dir=biosdk-client
 
-ENV work_dir_env=/
+ENV biosdk_local_dir_name=${boisdk_local_dir}

```

Comment:
@mandeepdhiman123 please update the variable name boisdk_local_dir to biosdk_local_dir